src/m365/teams/commands/meeting/meeting-transcript-list.ts

```diff
@@ -0,0 +1,135 @@
+import auth from '../../../../Auth';
+import { Logger } from '../../../../cli/Logger';
+import GlobalOptions from '../../../../GlobalOptions';
+import GraphCommand from "../../../base/GraphCommand";
+import commands from '../../commands';
+import { odata } from '../../../../utils/odata';
+import { validation } from '../../../../utils/validation';
+import { accessToken } from '../../../../utils/accessToken';
+import { aadUser } from '../../../../utils/aadUser';
+import { MeetingTranscript } from '../../MeetingTranscript';
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  userId?: string;
+  userName?: string;
+  email?: string;
+  meetingId: string;
+}
+
+class TeamsMeetingTranscriptListCommand extends GraphCommand {
+  public get name(): string {
+    return commands.MEETING_TRANSCRIPT_LIST;
+  }
+
+  public get description(): string {
+    return 'Lists all transcripts for a given meeting';
+  }
+
+  public defaultProperties(): string[] | undefined {
+    return ['id', 'createdDateTime'];
+  }
+
+  constructor() {
+    super();
+
+    this.#initTelemetry();
+    this.#initOptions();
+    this.#initValidators();
+    this.#initOptionSets();
+  }
+
+  #initTelemetry(): void {
+    this.telemetry.push((args: CommandArgs) => {
+      Object.assign(this.telemetryProperties, {
+        userId: typeof args.options.userId !== 'undefined',
+        userName: typeof args.options.userName !== 'undefined',
+        email: typeof args.options.email !== 'undefined'
+      });
+    });
+  }
+
+  #initOptions(): void {
+    this.options.unshift(
+      {
+        option: '-u, --userId [userId]'
+      },
+      {
+        option: '-n, --userName [userName]'
+      },
+      {
+        option: '--email [email]'
+      },
+      {
+        option: '-m, --meetingId <meetingId>'
+      }
+    );
+  }
+
+  #initValidators(): void {
+    this.validators.push(
+      async (args: CommandArgs) => {
+        if (args.options.userId && !validation.isValidGuid(args.options.userId)) {
+          return `${args.options.userId} is not a valid Guid`;
+        }
+
```

Comment:
Let's also validate the userName and email using our validation util.

Reply:
@milanholemans so for upn we have a validator and I will add that but which validation you mean for email 🤔?
sorry for probably a stupid question 😅

Reply:
I'm not sure if we do this at other places as well, but the upn validator just checks if it's a valid email right?

Reply:
We could use the same validator as UPN to validate the email input.

Reply:
✅